charts/kong/values.yaml

```diff
@@ -183,8 +183,21 @@ plugins: {}
 # - kong-admin-tls
 secretVolumes: []
 
-# Set runMigrations to run Kong migrations
-runMigrations: true
+# Enable/disable migration jobs, and set annotations for them
+migrations:
+  # Enable initial migrations/"kong migrations bootstrap"
+  # This should be set to false after the initial install
+  init: true
+  # Enable pre-upgrade migrations/"kong migrations up"
+  preUpgrade: true
+  # Enable post-upgrade migrations/"kong migrations finish"
```

Comment:
The `/` is weird here. Can we use words instead so as to not throw off users?
Something along the lines:
Enable post-upgrade migrations (equivalent to "kong migrations finish")

This applies to other two instances as well.